src/test/java/JunitPracticalTask1.java

```diff
@@ -0,0 +1,67 @@
+import org.assertj.core.api.SoftAssertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.junit.jupiter.api.Assertions;
+
+import java.util.List;
+
+
+public class JunitPracticalTask1 {
+
+    @Test
+    public void equal() {
+        Assertions.assertEquals(1, 1);
+    }
+
+    @Test
+    public void notEqual_Test() {
+        Assertions.assertNotEquals(1, 2);
+    }
+
+    @Test
+    public void assertTrue_Test() {
+        Assertions.assertTrue(2 > 1);
+    }
+
+    @Test
+    public void assertFalse_Test() {
+        Assertions.assertFalse(2 < 1);
+    }
+
+    @Test
+    public void failTest() {
+        List<String> list = List.of(
+                "One",
+                "Two",
+                "Three");
+        for (String s : list) {
+            if (s.equals("")) {
+                Assertions.fail("Empty string in prohibited!");
+            }
+        }
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"one", "two", "three"})
+    public void paramTest(String value) {
+        Assertions.assertNotEquals("", value);
+    }
+
+    @Test
+    public void assertFalse_failTest() {
+        Assertions.assertFalse(2 > 1);
```

Comment:
A bit odd task to have a failing test... such a pipeline won't ever succeed
@stanislav, 
![image](https://user-images.githubusercontent.com/6898003/89608425-c5d7fe80-d87d-11ea-9cb9-34a0b9cc28ec.png)

Reply:
There's no task to create failed test. Student should make his existing test failed to verify that it actually tests something.